kythe/go/extractors/gcp/examples/mvn.yaml

```diff
@@ -35,6 +35,6 @@ artifacts:
   objects:
     location: 'gs://${_BUCKET_NAME}/'
     paths:
-      - '/workspace/out/javac-extractor.err'
       - '/workspace/out/test.kzip'
+      - '/workspace/out/javac-extractor.err'
```

Comment:
Revert this change unless it is meaningful

Reply:
I do actually want the output object first, since error there will abort attempts to upload the next thing.  The err output is less important to me.

(Ideally in the future we'll figure out how to get both up there in a more resilient fashion, but for now this is better imo)